### PR TITLE
feature (T32026) : remove unused attribute from handler

### PR DESCRIPTION
**Pull request by jalalifardemos**
**Tickets**: https://yaits.demos-deutschland.de/T32026

**Description**: Remove 'email2' from handler 

### Linked PRs (optional)

- https://github.com/demos-europe/demosplan-project-ewm/pull/86

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Logic/User/OrgaHandler.php [hunk 1]
```diff
@@ -159,7 +159,6 @@ public function getWritableAttributes(): array
             'ccEmail2',
             'copySpec',
             'cssvars',
-            'email2',
```
Comment (AlexMenschoid) [inline demosplan/DemosPlanCoreBundle/Logic/User/OrgaHandler.php:162]: Why do you think, this is no longer used? I still see the property on the Orga entity which to me suggests, that the attribute should also still be writable.

**Review by AlexMenschoid — CHANGES_REQUESTED** [review]

File: demosplan/DemosPlanCoreBundle/Logic/User/OrgaHandler.php (hunk 1 above)
Comment (jalalifardemos) [reply demosplan/DemosPlanCoreBundle/Logic/User/OrgaHandler.php:162]: In `DpOrganisationFormFields.vue:407-584` used **email2** as a condition and i should remove it to not show email field. (base on Florian's Comment )

Comment (jalalifardemos) [reply demosplan/DemosPlanCoreBundle/Logic/User/OrgaHandler.php:162]: what about [3250e47](https://github.com/demos-europe/demosplan-core/pull/1940/commits/3250e47382beef4c3f55e0e9e394c0196418d0e3)

**Review by AlexMenschoid — APPROVED** [review]
lgtm, code review only

**Pull request merged**